scripts/bundles/testing.ts

```diff
@@ -127,4 +127,17 @@ async function copyTestingInternalDts(opts: BuildOptions, inputDir: string) {
       return false;
     },
   });
+
+  // Replace `@stencil/core/declarations` imports with `@stencil/core/internal` imports in
+  // all typedef files
+  const files = await fs.readdir(opts.output.testingDir);
+  for (const file of files) {
+    if (file.endsWith('.d.ts')) {
+      const dts = await fs.readFile(join(opts.output.testingDir, file), 'utf8');
+      await fs.writeFile(
+        join(opts.output.testingDir, file),
+        dts.replace('@stencil/core/declarations', '@stencil/core/internal'),
+      );
+    }
+  }
```

Comment:
Needed to add this block to fix our bundled output after changing the import location in [this file](https://github.com/ionic-team/stencil/pull/4897/files#diff-b7ac49093915e41678ffa5479215ca14677f19f2d9bc2b16fcb2332c9c2e04a5). See comment there for more context.